frontend/src/app/Settings/Facility/facilitySchema.ts

```diff
@@ -73,6 +73,9 @@ const providerSchema: yup.SchemaOf<RequiredProviderFields> = yup.object({
 const deviceTypeSchema: yup.SchemaOf<DeviceType> = yup.object({
   internalId: yup.string().required(),
   name: yup.string().required(),
+  // TODO: This property does not represent user input, so it probably
+  // shouldn't be here -- for now, a shim to keep yup+TS compiler happy
+  testLength: yup.number().optional(),
```

Comment:
`testLength` isn't a property set by the facility form, so I don't think it's meaningful to include it in our validation - but since it's on the `DeviceType` interface that the `yup` schema is building from, I had to account for it somehow.

I had expected I could type this schema as `yup.schemaOf<Omit<DeviceType, "testLength">>` but we still get a TS error 🤔 

The correct thing to do, even if it's annoying, might be to split that interface into `DeviceType` and `DeviceTypeInput` and then doing the same with `DeviceSpecimenType` to account for the differences in shape.